Closes #63

JoshuaAaron opened the pull request:
Added the OrcaStars website logo, about us information, email, and youtube links

JoshuaAaron committed:
Closes #63

Added the OrcaStars website logo, about us information, email, and youtube links

charlesmsy reviewed (changes requested):
Could we also get the "AboutUsFragment" with this?

VBeegle reviewed (changes requested):
This pull request commit has not added the missing AboutUsFragment and corresponding layout. Please make sure to add them to your commit before merging

JoshuaAaron committed:
Closes #63

Added the OrcaStars website logo, about us information, email, and youtube links

JoshuaAaron commented:
Oh wow, I totally missed that it didn't add those files. It should good to go now. Sorry team

JoshuaAaron committed:
Delete Gradle__androidx_annotation_annotation_1_0_0_jar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_appcompat_appcompat_1_0_2_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_arch_core_core_common_2_0_0_jar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_arch_core_core_runtime_2_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_asynclayoutinflater_asynclayoutinflater_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_collection_collection_1_0_0_jar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_coordinatorlayout_coordinatorlayout_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_core_core_1_0_1_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_cursoradapter_cursoradapter_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_customview_customview_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_documentfile_documentfile_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_drawerlayout_drawerlayout_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_fragment_fragment_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_interpolator_interpolator_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_legacy_legacy_support_core_ui_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete workspace.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_legacy_legacy_support_core_utils_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_legacy_legacy_support_v4_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_lifecycle_lifecycle_common_2_0_0_jar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_lifecycle_lifecycle_livedata_2_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_lifecycle_lifecycle_livedata_core_2_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_lifecycle_lifecycle_runtime_2_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_lifecycle_lifecycle_viewmodel_2_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_loader_loader_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_localbroadcastmanager_localbroadcastmanager_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_media_media_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_print_print_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_slidingpanelayout_slidingpanelayout_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_swiperefreshlayout_swiperefreshlayout_1_0_0_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_test_espresso_espresso_core_3_1_1_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_test_espresso_espresso_idling_resource_3_1_1_aar.xml

deleting the Map Test files that were  uploaded by mistake

JoshuaAaron committed:
Delete Gradle__androidx_test_monitor_1_1_1_aar.xml

JoshuaAaron committed:
Delete Gradle__androidx_test_runner_1_1_1_aar.xml

JoshuaAaron committed:
Delete Gradle__androidx_vectordrawable_vectordrawable_1_0_1_aar.xml

JoshuaAaron committed:
Delete Gradle__androidx_vectordrawable_vectordrawable_animated_1_0_0_aar.xml

JoshuaAaron committed:
Delete Gradle__androidx_versionedparcelable_versionedparcelable_1_0_0_aar.xml

JoshuaAaron committed:
Delete Gradle__androidx_viewpager_viewpager_1_0_0_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_android_gms_play_services_base_16_0_1_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_android_gms_play_services_basement_16_0_1_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_android_gms_play_services_location_16_0_0_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_android_gms_play_services_maps_16_1_0_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_android_gms_play_services_places_placereport_16_0_0_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_android_gms_play_services_tasks_16_0_1_aar.xml

JoshuaAaron committed:
Delete Gradle__com_google_code_findbugs_jsr305_2_0_1_jar.xml

JoshuaAaron committed:
Delete Gradle__com_squareup_javawriter_2_1_1_jar.xml

JoshuaAaron committed:
Delete Gradle__javax_inject_javax_inject_1_jar.xml

JoshuaAaron committed:
Delete Gradle__junit_junit_4_12_jar.xml

JoshuaAaron committed:
Delete Gradle__net_sf_kxml_kxml2_2_3_0_jar.xml

JoshuaAaron committed:
Delete modules.xml

JoshuaAaron committed:
Delete Gradle__org_hamcrest_hamcrest_core_1_3_jar.xml

JoshuaAaron committed:
Delete Gradle__org_hamcrest_hamcrest_integration_1_3_jar.xml

JoshuaAaron committed:
Delete Gradle__org_hamcrest_hamcrest_library_1_3_jar.xml
